### PR TITLE
Add configurable max tasks per updater  options to diagnostic_updater

### PR DESCRIPTION
**Pull request by bjv-capra**
Signed-off-by: Bart Jimenez Vera <bjv@capra.ooo>

### CODE REVIEW

File: micro_ros_diagnostic_updater/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.config
```
Comment (bjv-capra) [inline micro_ros_diagnostic_updater/.gitignore:1]: Need to remove this

**Comment by bjv-capra** [comment]
Closes #29

File: micro_ros_diagnostic_updater/.gitignore (hunk 1 above)
Comment (norro) [reply micro_ros_diagnostic_updater/.gitignore:1]: agreed, this shouldn't be necessary

**Review by norro — APPROVED** [review]
LGTM.
Thanks for the PR!

**Pull request merged**